aimmo-game/service.py

```diff
@@ -4,9 +4,8 @@
 import sys
 
 # If we monkey patch during testing then Django fails to create a DB enironment
```

Comment:
This will now monkey patch always. Is the above comment now invalid, or do we still need to avoid monkey patching during testing?